src/containers/CreateInstructor/index.jsx

```diff
@@ -1,21 +1,34 @@
 import { connect } from 'react-redux';
 
 import CreateInstructorPage from '../../components/CreateInstructorPage';
-import { createInstructor } from '../../data/actions/instructorInfo';
+import { fetchInstructorInfo, createInstructor } from '../../data/actions/instructorInfo';
 import { fetchInstructorOptions } from '../../data/actions/instructorOptions';
 
 const mapStateToProps = state => ({
-  publisherUserInfo: state.publisherUserInfo,
+  publisherUserInfo: state.publisherUserInfo,  // BJH: can we ditch this?
```

Comment:
You should be good to get rid of this since it's not used on the creation page

Reply:
Done.